From 9e5e1ef431b97839f026aa106dbc64ddf456de3f Mon Sep 17 00:00:00 2001
From: AKINNUOYE TIMOTHY <akinnuoyet@gmail.com>
Date: Wed, 19 Jun 2024 02:50:40 +0100
Subject: [PATCH] Ensure all files are present in restore-point

---
 backend/main.py | 46 +---------------------------------------------
 1 file changed, 1 insertion(+), 45 deletions(-)

diff --git a/backend/main.py b/backend/main.py
index 518c3c2..c4fb89e 100755
--- a/backend/main.py
+++ b/backend/main.py
@@ -1,7 +1,6 @@
-from fastapi import FastAPI
 import uvicorn
 import logging
-from fastapi import FastAPI, Request, Response
+from fastapi import FastAPI, Request
 from fastapi.middleware.cors import CORSMiddleware
 from backend.routers import users, auth, countries, institution
 from backend.database import engine, Base
@@ -11,24 +10,16 @@ Base.metadata.create_all(bind=engine)
 app = FastAPI()
 
 origins = [
-<<<<<<< HEAD
     "http://localhost",
     "http://localhost:8000",
     "http://127.0.0.1:5500",
-=======
-    "http://127.0.0.1:5500",
     "https://mole-relevant-salmon.ngrok-free.app",
     "*"
->>>>>>> made few changes
 ]
 
 app.add_middleware(
     CORSMiddleware,
-<<<<<<< HEAD
     allow_origins=origins,
-=======
-    allow_origins=origins,  # Or list specific origins (e.g., your frontend URL)
->>>>>>> made few changes
     allow_credentials=True,
     allow_methods=["*"],  # Make sure to include "OPTIONS"
     allow_headers=["*"],
@@ -61,41 +52,6 @@ app.include_router(institution.router, prefix="/api/institution", tags=["institu
 @app.get("/")
 def read_root():
     return {"message": "Welcome to the API"}
-<<<<<<< HEAD
-from fastapi import FastAPI
-from fastapi.middleware.cors import CORSMiddleware
-from backend.routers import users, auth
-from backend.database import engine, Base
-
-Base.metadata.create_all(bind=engine)
-
-
-app = FastAPI()
-
-origins = [
-    "http://localhost",
-    "http://localhost:8000",
-    "http://127.0.0.1:5500",
-]
-
-app.add_middleware(
-    CORSMiddleware,
-    allow_origins=origins,
-    allow_credentials=True,
-    allow_methods=["*"],
-    allow_headers=["*"],
-)
-
-
-app.include_router(users.router, prefix="/api/users", tags=["users"])
-app.include_router(auth.router, prefix="/api/auth", tags=["auth"])
-
-
-@app.get("/")
-def read_root():
-    return {"message": "Welcome to the API"}
-=======
 
 if __name__ == "__main__":
     uvicorn.run(app, host="0.0.0.0", port=8000)
->>>>>>> made few changes
-- 
2.14.1

